lesson2/homework.js

```diff
@@ -0,0 +1,30 @@
+
+
+const addNewTask = function(id, title, description, created, updated, dueDate, statusID, userID) {
+  
+};
+
+const changeTaskTitle = function(taskID, newTitle) {
+  
+};
+
+const changeTaskDueDate = function(taskID, newDueDate) {
+  
+};
+
+const changeTaskStatus = function(taskID, newStatus) {
+  
+};
+
+const markTaskAsCompleted = function(taskID, newStatus) {
```

Comment:
I guess the function would always use the same status ("Done") so i don't think we need the newStatus argument

Reply:
true!